src/index.html

```diff
@@ -1,22 +1,204 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1.0"
-    />
-    <title>Landing Page</title>
-    <link
-      rel="stylesheet"
-      href="./styles/main.scss"
-    />
-    <script
-      src="scripts/main.js"
-      defer
-    ></script>
-  </head>
-  <body>
-    <h1>Landing Page</h1>
-  </body>
+
+
+<head>
+
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Landing Page</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Manrope:wght@200..800&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="./styles/main.scss" />
+  <script src="scripts/main.js" defer></script>
+
+</head>
+
+<body>
+
+  <header class="header">
+
+    <div class="header__content">
+      <div class="top-bar">
+        <a href="#" class="top-bar__logo-link">
+          <img class="top-bar__logo" src="images/logo.png" alt="Logo">
+        </a>
+
+        <div class="top-bar__icons">
+          <a href="tel:+12345555-55-55" class="icon icon--phone"></a>
+          <a href="#menu" class="icon icon--menu"></a>
+        </div>
+      </div>
+
+      <div class="bottom">
+        <h1 class="bottom__title">Your senses <br /> Set free.</h1>
+      </div>
+    </div>
+
+    <aside class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo-link">
+            <img src="images/logo.png" alt="Bang & olufsen logo" class="top-bar__logo">
+          </a>
+
+          <a href="#" class="icon icon--close"></a>
+        </div>
+
+        <div class="menu__bottom">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a class="nav__link">Recommended</a>
```

Comment:
Menu links should be actual links to their relevant page section [Reference](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md#:~:text=Ensure%20that%20all%20nav__links%20work%20smoothly)